openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -0,0 +1,20 @@
+""" Views that we will use to view toggle state in edx-platform. """
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+
+from rest_framework.authentication import SessionAuthentication
+from rest_framework import permissions, views
+from rest_framework.response import Response
+
+from openedx.core.lib.api.permissions import IsStaffOrOwner
+from openedx.core.lib.api.authentication import BearerAuthentication
+
+
+class ToggleStateView(views.APIView):
+    """
+    An endpoint for displaying the state of toggles in edx-platform.
+    """
+    authentication_classes = (JwtAuthentication, BearerAuthentication, SessionAuthentication,)
+    permission_classes = (permissions.IsAuthenticated, IsStaffOrOwner,)
```

Comment:
Let's use `IsStaff` from here, rather than `IsStaffOrOwner`: https://github.com/edx/edx-drf-extensions/blob/master/edx_rest_framework_extensions/permissions.py#L26